doc/level1.rst

```diff
@@ -550,9 +556,9 @@ with more than one recipient. These headers MUST be placed in the root
 MIME part of the encrypted message payload. The encrypted payload in
 this case contains one Autocrypt-Gossip header for each recipient,
 which MUST include ``addr`` and ``keydata`` attributes with the
-relevant data from the sender's Autocrypt :ref:`peer state
-<peer-state>` about the recipient. It SHOULD NOT contain a
-``prefer-encrypt`` attribute.
+corresponding values for the recipient identified by ``addr`` as
```

Comment:
shouldn't this be ``to-addr`` ?

Reply:
So, Bob receives a message from Alice, and Carol is in cc.  In this case, Bob probably thinks about his address as being the ``to-addr``.  I think it is okay to use ``addr`` in this case.  Perhaps ``other-addr`` would be better, but I think ``to-addr`` is confusing.

Reply:
maybe use the role of the address rather than its originating header, as in `rcpt-addr`?